### PR TITLE
[4.1] Jooa11y updates

### PR DESCRIPTION
**Pull request by brianteeman**
Various updates from upstream including addressing a multiples issue with language files

- [x] rel="noopener noreferrer" #37014 thanks @tecpromotion
- [x] new tab warning tip #37048 thanks @particthistle
- [x] remove bookmarklet from the repo
- [x] link in readme
- [x] fix escape key not working - thanks @Fedik
- [x] add alt+s keyboard shortcut for skip to next issue thanks @adamchaboryk
- [x] add alt+a keyboard shortcut to open panel thanks @adamchaboryk
- [x] update doc for keyboard shortcuts
- [x] Nudge buttons if they overlap. thanks @adamchaboryk
- [x] Licence change upstream to gpl
- [x] new test from upstream - check for alt and caption
- [x] update panel messages to make more l13n friendly
- [x] fake headings
- [x] Link Check Advanced 
- [x] Alt text check
- [x] CSS updates
- [x] new test from upstream - check duplicate id
- [x] new test from downstream - check underline 
- [x] use version published on npm - thanks @bembelimen

### CODE REVIEW

**Comment by brianteeman** [comment]
This is needed to resolve the drone issues

File: plugins/system/jooa11y/jooa11y.xml [hunk 1]
```diff
@@ -2,12 +2,12 @@
 <extension type="plugin" group="system" method="upgrade">
 	<name>plg_system_jooa11y</name>
 	<author>Joomla! Project</author>
-	<creationDate>November 2021</creationDate>
+	<creationDate>February 2022</creationDate>
 	<copyright>(C) 2021 Open Source Matters, Inc.</copyright>
```
Comment (laoneo) [inline plugins/system/jooa11y/jooa11y.xml:6]:
```suggestion
	<copyright>(C) 2022 Open Source Matters, Inc.</copyright>
```

File: plugins/system/jooa11y/jooa11y.xml [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
 	<authorEmail>admin@joomla.org</authorEmail>
 	<authorUrl>www.joomla.org</authorUrl>
-	<version>4.1.0</version>
+	<version>4.2.0</version>
```
Comment (laoneo) [inline plugins/system/jooa11y/jooa11y.xml:10]:
```suggestion
	<version>__DEPLOY_VERSION__</version>
```

Comment (brianteeman) [reply plugins/system/jooa11y/jooa11y.xml:10]: its the version of the plugin not of joomla

File: plugins/system/jooa11y/jooa11y.xml (hunk 1 above)
Comment (brianteeman) [reply plugins/system/jooa11y/jooa11y.xml:6]: copyright is from the first date. This is consistent with things like the language files
image

**Comment by brianteeman** [comment]
Now uses the version published at npm

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
Awesome - thanks

**Pull request merged**